src/App.svelte

```diff
@@ -77,7 +79,9 @@
       method: 'POST'
     }
       let usersAddedBook = bookInfo
-      fetch(`http://book-basket-be.herokuapp.com/create_book?author=${usersAddedBook.authors.data[0].attributes.name}&genre=${usersAddedBook.genres.data[0].attributes.type}&title=${usersAddedBook.title}&isbn=${usersAddedBook.isbn}&date_published=${usersAddedBook.published_date}&summary=${usersAddedBook.summary}&image_url=${usersAddedBook.image_url}`, header)
+      var uri = `http://book-basket-be.herokuapp.com/create_book?author=${usersAddedBook.authors.data[0].attributes.name}&genre=${usersAddedBook.genres.data[0].attributes.type}&title=${usersAddedBook.title}&isbn=${usersAddedBook.isbn}&date_published=${usersAddedBook.published_date}&summary=${usersAddedBook.description}&image_url=${usersAddedBook.image_url}`;
```

Comment:
I think we also need to encode this as well `usersAddedBook.image_url`, since we need to escape out of some of the characters in URLs to pass them through completely